android/default.nix

```diff
@@ -94,6 +94,12 @@ in rec {
 
     , additionalDependencies ? ""
 
+    , nativeDependencies ? {}
```

Comment:
Please rename this. It's a fine name in isolation, but conflicts with `nativeBuildInputs` which is *build* time (not on the phone) dependencies while `buildInputs` is run-time (on the phone) dependencies.

Reply:
Also, why isn't the default value `_: {}` rather than `{}`? Doesn't that better match the example?

Reply:
So I suggest next try :) `runtimeSharedLibs`